site/src/docs/components/status-label/index.mdx

```diff
@@ -36,28 +36,14 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 
 ### Variations
 
-#### Default
+#### Rectangular
 
-StatusLabel comes in five (5) different types. One neutral and four (4) with different status colours.
-
-<PlaygroundPreview>
-  <StatusLabel>Neutral</StatusLabel>
-  <StatusLabel type="info" style={{ marginLeft: 'var(--spacing-s)' }}>
-    Info
-  </StatusLabel>
-  <StatusLabel type="success" style={{ marginLeft: 'var(--spacing-s)' }}>
-    Success
-  </StatusLabel>
-  <StatusLabel type="alert" style={{ marginLeft: 'var(--spacing-s)' }}>
-    Alert
-  </StatusLabel>
-  <StatusLabel type="error" style={{ marginLeft: 'var(--spacing-s)' }}>
-    Error
-  </StatusLabel>
-</PlaygroundPreview>
+<StatusLabel variant="rounded" type="alert">Deprecated</StatusLabel>
```

Comment:
I think you can still keep the Default heading and the examples here because they are still usable, just not advisable to use. The notification at the top of the page and this Deprecated label will inform the user of what is exactly deprecating and changing.